### ISSUE TITLE
With Chrome >= 58.x uBlock can handle FULLY itself all websocket "things"?

### ISSUE THREAD

**Issue by Alpengreis** [gorhill/uBlock#2478]
Am I right that with Chrome >= 58.x uBlock can handle FULLY itself all websocket "things" **and I do no more need uBlock Extra**?

**Comment by mapx-** [comment]
probably "WebSocket opened from the SharedWorker" wont be blocked.
see
https://issues.adblockplus.org/ticket/4770#comment:8

but @gorhill could clear better this question

**Comment by Alpengreis** [comment]
Thank you for the link so far, interesting!

**Comment by gorhill** [comment]
Yes, starting with Chromium 58+ and uBO 1.11.5+, websocket connection attempts will be filtered by uBO itself.

However uBO Extra has become multipurpose, and whether you still need it depends on whether you are a visitor to one of the sites for which uBO-Extra contains special fixes.

@mapx- this is a different issue, this has been addressed in uBO itself with 1.11.2, related issue: <https://github.com/gorhill/uBlock/issues/2360>. Solution is to prevent the shared worker to be created in the first place using a `|blob:` filter to decide whether a CSP directive to restrict `blob:`-based shared workers.

**Comment by gorhill** [comment]
@Alpengreis can you still confirm that the warning you get in <https://github.com/gorhill/uBlock/issues/2435> **do not** break uBO at all? This is very important. If it's only warnings, then this is an acceptable side effect of supporting ability to block websockets as early as when Chromium 58 is released.

**Comment by Hrxn** [comment]
> However uBO Extra has become multipurpose, and whether you still need it depends on whether you are a visitor to one of the sites for which uBO-Extra contains special fixes.

Is there a list of these sites, to get an overview?

**Comment by okiehsch** [comment]
They are inside `contentscript.js` for example sites using `Instart Logic`
https://github.com/gorhill/uBO-Extra/blob/master/contentscript.js#L272

**Comment by Hrxn** [comment]
Ah, okay. Thanks. I expected something along the lines of "filters.txt" or "sites.txt" somewhere, not directly in 'contentscript.js'

Well, from the listed sites I only visit pcmag.com semi-regularly, but I don't see any ads there, without uBO-Extra. Is this location-based maybe?

**Comment by gorhill** [comment]
[Sites on which uBO-Extra is useful](https://github.com/gorhill/uBO-Extra/wiki/Sites-on-which-uBO-Extra-is-useful).

**Comment by Alpengreis** [comment]
**@gorhill** 

Shortly thereafter, I uninstalled Opera release version. Then I had few days no Chrome and now Chrome Beta (v58.0.3029.33 beta 64-bit), even because that websocket integration (my Browser to really work with is still Fx (now in Developer version).

I had tested that only short anyway and saw it does still block. I had not tested it more. The last version I saw, that uBlock do still block was (without guarantee) perhaps v1.11.5b1 ...
